float32 est_vel
